src/test/setup.ts

```diff
@@ -29,7 +29,7 @@ const ignoreConsoleErrorMessages = [
 
 const originalConsoleError = console.error;  // tslint:disable-line:no-console
 console.error = (msg: string) => {  // tslint:disable-line:no-console
```

Comment:
I think your change to this type declaration which we've discussed yesterday didn't get pushed